core/src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -943,7 +943,9 @@ public void onPlayerDamage(final EntityDamageEvent event) {
 				if (bPlayer.getBoundAbilityName().equalsIgnoreCase("Shockwave")) {
 					new Shockwave(player, true);
 				} else if (bPlayer.getBoundAbilityName().equalsIgnoreCase("Catapult")) {
-					new EarthPillars(player, true);
+					if (CoreAbility.getAbility(EarthPillars.class).isEnabled()) {
```

Comment:
I'd actually just check if `"Abilities.Earth.EarthPillars.Enabled"` is true in `ConfigManager`. It saves having to call `CoreAbility#getAbility` when checking if it is enabled in the config is effectively the same.

Additionally, I'm not sure if the ability being disabled on the player's end (i.e., player doesn't have permissions to use `EarthPillars`) also creates the error, but if so, add a condition to see if the player can bend `EarthPillars` prior to instantiation as well. If not, ignore this!